.github/workflows/deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy Staging
+
+on:
+  push:
+    branches:
+      - pm2
```

Comment:
should this be changed now that we're merging pm2?

Reply:
good catch